shallow_backup/config.py

```diff
@@ -78,46 +79,70 @@ def safe_create_config():
 		write_config(backup_config)
 
 
-# TODO: Rethink these methods.
-def add_to_config(section, path):
+def add_to_config(section, path, dest=None):
 	"""
 	Adds the path under the correct section in the config file.
-	FIRST ARG: [dot, config, other]
+	FIRST ARG: [dot, config]
 	SECOND ARG: path, relative to home directory for dotfiles, absolute for configs
 	"""
-	full_path = home_prefix(path)
-	if not os.path.exists(full_path):
-		print_red_bold("ERR: {} doesn't exist.".format(full_path))
-		sys.exit(1)
-
 	if section == "dot":
+		# Check if the file is in the home directory.
+		full_path = home_prefix(path)
+		if not os.path.exists(full_path):
+			print_red_bold("ERR: {} doesn't exist.".format(full_path))
+			sys.exit(1)
+
 		# Make sure dotfile starts with a period
 		if path[0] != ".":
 			print_red_bold("ERR: Not a dotfile.")
 			sys.exit(1)
 
 		if not os.path.isdir(full_path):
 			section = "dotfiles"
-			print(Fore.BLUE + Style.BRIGHT + "Adding {} to dotfile backup.".format(full_path) + Style.RESET_ALL)
+			print_blue_bold("Adding {} to dotfile backup.".format(full_path))
 		else:
 			section = "dotfolders"
 			if path[-1] != "/":
 				full_path += "/"
 				path += "/"
-			print(Fore.BLUE + Style.BRIGHT + "Adding {} to dotfolder backup.".format(full_path) + Style.RESET_ALL)
+			print_blue_bold("Adding {} to dotfolder backup.".format(full_path))
 
-	# TODO: Add config section once configs backup prefs are moved to the config file
 	elif section == "config":
-		print(Fore.RED + Style.BRIGHT + "ERR: Option not currently supported." + Style.RESET_ALL)
-		sys.exit(1)
-	elif section == "other":
-		print(Fore.RED + Style.BRIGHT + "ERR: Option not currently supported." + Style.RESET_ALL)
+		# TODO: Finish proofing this for which abs/non-abs paths get printed and stored.
```

Comment:
Finish proofing this method for which abs/non-abs paths get printed and stored. 

<img width="1052" alt="image" src="https://user-images.githubusercontent.com/20600565/47615059-f15a7200-da77-11e8-9309-f1afd67a558f.png">


This shouldn't happen.